packages/main/src/backend/import/importTransactions.ts

```diff
@@ -97,26 +92,6 @@ function emitChromeDownload(eventPublisher: EventPublisher, percent: number) {
   eventPublisher.emit(EventNames.DOWNLOAD_CHROME, new DownalodChromeEvent(percent));
 }
 
```

Comment:
What is it? I see it is unused..?

If it is not related to your work, I prefer you to open a new PR for this, so if there will be a problem with this remove, it will be easy to revert it.